src/clojure_mcp/file_content.clj

```diff
@@ -21,8 +21,26 @@
 (def ^MediaTypeRegistry registry
   (.getMediaTypeRegistry (MimeTypes/getDefaultMimeTypes)))
 
-(defn text-media-type? [mime]
-  (.isInstanceOf registry (MediaType/parse mime) MediaType/TEXT_PLAIN))
+(def text-like-mime-patterns
+  "Regex patterns for MIME types that should be treated as text.
+   Limited to SQL, JSON, YAML, and XML formats that may not inherit 
+   from text/plain in the Apache Tika MediaType hierarchy."
+  [#"application/sql"
+   #"application/json"
+   #"application/.*xml"
+   #"application/.*yaml"])
```

Comment:
I wonder if this vector of patterns would be best expressed via the `config.edn` 🤔 

Maybe it's sufficient to let the list of cases that are _not_ supported by the the Tika MediaType hierarchy simply grow according to need within `text-like-mime-patterns`